psl/builtin-connectors/src/mysql_datamodel_connector/validations.rs

```diff
@@ -106,12 +107,14 @@ pub(crate) fn uses_native_referential_action_set_default(
     };
 
     let warning_msg = || {
-        format!(
-            "Using {set_default} on {connector} may yield to unexpected results, as the database will silently change the referential action to `{no_action}`.",
+        formatdoc!(
+            r#"
+            `{connector_name}` does not actually support the `{set_default}` referential action, so using it may result in unexpected errors.
```

Comment:
The snapshot indicates that this uses the full database name, not anything reflected in code, so I would say the backticks can go:
```suggestion
            {connector_name} does not actually support the `{set_default}` referential action, so using it may result in unexpected errors.
```

Reply:
Fair enough